### DISCUSSION TITLE
Gremlins unable to download modules

### DISCUSSION THREAD

**Discussion by d4n13lg4rc14** [go-gremlins/gremlins#235] [Q&A]
Hello, I have been using Gremlins for awhile without problems. Since last week, I start having a error message when gremlins try to download module to gather the coverage:

"failed to gather coverage: impossible to download modules: exit status 1"

I was working with the version 0.4.0 before, since the issue started I have bumped to the latest version 0.5.0, hoping something would change, however, the issue persists.

I have not made any changes on the project itself or in the gremlins config. I have the following config:
"
---
unleash:
  silent: false
  integration: false
  dry-run: false
 coverpkg: "./internal/..."
 threshold:
    efficacy: 99.99
...
"

Any insights or help would be greatly appreciated. Thank you

**Comment by k3rn31** [comment]
Hi Daniel, it's hard to say what's failing... 
I'd try a git bisect to understand when this problem begins. 
Also, you can try to run gremlins in debug to see exactly what command is used when downloading modules and try to use that same command from the shell.

**Reply by d4n13lg4rc14** [reply]
I checked the docs and I did not find the flag/command to run gremlins on "debug mode". How should it be done? Thank you

**Comment by k3rn31** [comment]
I mean to run it in debug from the source code.
